pkg/protobundle/bundle.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protobundle
+
+import (
+	"fmt"
+
+	pbcommon "github.com/sigstore/protobuf-specs/gen/pb-go/common/v1"
+	pbrekor "github.com/sigstore/protobuf-specs/gen/pb-go/rekor/v1"
+	"github.com/sigstore/rekor/pkg/generated/models"
+
+	"github.com/sigstore/rekor/pkg/tle"
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+)
+
+// GenerateTransparencyLogEntry returns a sigstore/protobuf-specs compliant message containing a
+// TransparencyLogEntry as defined at https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_rekor.proto
+func GenerateTransparencyLogEntry(anon models.LogEntryAnon) (*pbrekor.TransparencyLogEntry, error) {
+	return tle.GenerateTransparencyLogEntry(anon)
```

Comment:
@haydentherapper currently this still has the log index bug fixed in https://github.com/sigstore/rekor/pull/1599 because of the pinned version of rekor imported. Do we need to do a rekor release to update the version? Or can I just pin the most recent commit? Or should I just wrap this call in the fix as well for the time being?

Reply:
We can just update the version of Rekor in go.mod - `go get github.com/sigstore/rekor@96dad3c1f9e210514858fcb2473268aab3aff69a` for the latest commit. When we cut a new release, dependabot will update the rekor dependency to the latest release